platform-api/che-core-api-builder/src/test/java/org/eclipse/che/api/builder/BuilderTest.java

```diff
@@ -95,6 +102,19 @@ public boolean removeListener(SourceManagerListener listener) {
                 }
             };
         }
+
+        @Override
+        public Map<String, BuilderEnvironment> getEnvironments() {
+            Map<String, BuilderEnvironment> env = new HashMap<>();
+
+            BuilderEnvironment builderEnvironment = DtoFactory.getInstance().createDto(BuilderEnvironment.class)
```

Comment:
you can use newDto(BuilderEnvironment.class)  (importing static newDto method from DtoFactory)